src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
```

Comment:
- [ ] Please, this component (App is a component too) make it so it is a class based component. That way we will to meet the project requirements.

## Explanation

In order to meet the requirements, all the components should be class-based.  `App` is a component too.